composer.json

```diff
@@ -13,12 +13,13 @@
         }
     ],
     "require": {
-        "php": ">=5.3.9",
+        "php": ">=5.5.0",
```

Comment:
PHP 5.5 is EOL now too. Everyone should be on PHP 5.6+ unless they are making security fixes and building PHP themselves.